src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,72 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    public String getSalaryInfo(String[] names, String[] data,
+                                String dateFrom, String dateTo) {
+        return getDrawResultSalaries(names, data, dateFrom, dateTo);
```

Comment:
What is the point to have method that just returns result of other method?
Place here code from getDrawResultSalaries() and remove redundant method